parm/atm/common/c384.yaml

```diff
@@ -0,0 +1,9 @@
+io_layout_x: 1
```

Comment:
I do not know how this file will be used.  Do we want all the parameters in this file hardwired or should some be variables?   For example, the anl and ges np variables won't be the same for a dual resolution setup (e.g., operational gdas).

Reply:
The idea was to have this fill in resolution-specific variables to then put in other places in the final YAML. But based on this and other comments, I am feeling that you are suggesting some sort of pre-processing script that takes `$CASE`, `$CASE_ENKF` and `$DOHYBVAR` (among others) and produce this and the analysis/background YAML sections on demand.

Reply:
For the time being it's ok to set variables here.  I'm looking ahead to when we release this to developers.   They will prefer setting variables in a central high level location and then having pre-processing scripts set up lower level stuff based on their high level input.   `$CASE`, `$CASE_ENKF`, `$DOYBVAR` currently exist at the g-w level.   These and other variables are both g-w variables and GDASapp variables.   We don't want to require developers install and build g-w if they only want to run GDASapp.   We also don't want to require developers set some variables for parallels at the g-w level and other variables at the GDASapp level.   Can GDASapp setup scripts perform a hierarchal search of a dictionary to set variables which may be defined at the g-w or GDASapp level?   Maintaining such a dictionary could be problematic since we need to  keep tabs of multiple repos.

Reply:
The only thing I'm really worried about is `layout_x` and `layout_y`. Those are easily set as part of `config.resources` in the g-w. Since that has to correspond with MPI tasks, is it reasonable to expect a user running the scripts to set their own `layout_x` and `layout_y` based on how many tasks they request?

For the `npx`, `npy`, etc. it should be easy enough to just pass `CASE`, `CASE_ENKF` and `DOHYBVAR` to the python `genYAML` code and let it compute all of these YAML keys.

Reply:
My mistake,  I mean in `config.fv3` . So those can easily come from there. But where are those stored in the GDASApp? I guess we could have a C96.yaml that just contains `layout_x` and `layout_y` and only looks for that if `$layout_x` is not set?

Reply:
Users will need to set their own `layout_x` and `layout_y` when submitting jobs at the GDASapp level.   If they are running the g-w, `config.fv3` can provide the definition.    As I have learned if `layout_x` and `layout_y` don't match with what I specify in my driver script `fv3jedi_var.x` notes this and aborts.

Reply:
Ok that works for me, I will work on a solution that removes some of these redundant YAML entries and push as part of a new commit to this PR. I have an idea of how to do so that I think should work fairly well.